tasktiger/__init__.py

```diff
@@ -34,503 +18,13 @@
-
-        This will keep workers from processing tasks for this queue until
-        the timeout has expired. Active tasks will continue processing their
-        current task.
-
-        timeout is number of seconds to hold the lock
-        """
-
-        key = self._key(LOCK_REDIS_KEY, queue)
-        Semaphore.set_system_lock(self.connection, key, timeout)
-
-    def get_queue_stats(self):
-        """
-        Returns a dict with stats about all the queues. The keys are the queue
-        names, the values are dicts representing how many tasks are in a given
-        status ("queued", "active", "error" or "scheduled").
-
-        Example return value:
-        { "default": { "queued": 1, "error": 2 } }
-        """
-
-        states = (QUEUED, ACTIVE, SCHEDULED, ERROR)
-
-        pipeline = self.connection.pipeline()
-        for state in states:
-            pipeline.smembers(self._key(state))
-        queue_results = pipeline.execute()
-
-        pipeline = self.connection.pipeline()
-        for state, result in zip(states, queue_results):
-            for queue in result:
-                pipeline.zcard(self._key(state, queue))
-        card_results = pipeline.execute()
-
-        queue_stats = defaultdict(dict)
-        for state, result in zip(states, queue_results):
-            for queue in result:
-                queue_stats[queue][state] = card_results.pop(0)
-
-        return queue_stats
-
-
-@click.command()
-@click.option(
-    '-q',
-    '--queues',
-    help=(
-        'If specified, only the given queue(s) are processed. Multiple queues '
-        'can be separated by comma.'
-    ),
-)
-@click.option(
-    '-m',
-    '--module',
-    help=(
-        'Module(s) to import when launching the worker. This improves task '
-        'performance since the module doesn\'t have to be reimported every '
-        'time a task is forked. Multiple modules can be separated by comma.'
-    ),
-)
-@click.option(
-    '-e',
-    '--exclude-queues',
-    help=(
-        'If specified, exclude the given queue(s) from processing. Multiple '
-        'queues can be separated by comma.'
-    ),
-)
-@click.option(
-    '-M',
-    '--max-workers-per-queue',
-    help='Maximum workers allowed to process a queue',
-    type=int,
-)
-@click.option(
-    '--store-tracebacks/--no-store-tracebacks',
-    help='Store tracebacks with execution history',
-    default=None,
-)
-@click.option('-h', '--host', help='Redis server hostname')
-@click.option('-p', '--port', help='Redis server port')
-@click.option('-a', '--password', help='Redis password')
-@click.option('-n', '--db', help='Redis database number')
-@click.pass_context
-def run_worker(context, host, port, db, password, **kwargs):
-    conn = redis.Redis(
-        host, int(port or 6379), int(db or 0), password, decode_responses=True
-    )
-    tiger = context.obj or TaskTiger(setup_structlog=True, connection=conn)
-    tiger.run_worker(**kwargs)
+from ._internal import *
```

Comment:
Imports should go above `__all__`.

Reply:
Not according to [PEP8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names). I've actually had an issue open in `isort` for quite a while now https://github.com/timothycrosley/isort/issues/711